locales.json

```diff
@@ -851,6 +851,22 @@
       "currentlyNoEvents": "Güncel bir etkinlik yok."
     }
   },
+  "news": {
+    "de": {
+      "news": "Nachrichten",
+      "newsAlternative": "Neuigkeiten",
+      "localNews": "Lokale Nachrichten",
+      "localNewsAlternative": "Lokale Neuigkeiten",
+      "readMore": "Weiterlesen",
+      "currentlyNoNews": "Aktuell gibt es keine Nachrichten.",
+      "currentlyNoNewsAlternative": "Aktuell gibt es keine Neuigkeiten."
+    },
+    "ar": {
+      "local": "الأخبار المحلية",
+      "currentlyNoNews": "ليست هناك أخبار متوفرة حاليا",
+      "readMore": "المزيد"
```

Comment:
@maxammann should we ask kiitos to update these translations or should we remove them and let it do by tolingo?

Reply:
Lets remove it I'd say, that way we have consistent quality. Maybe worse, but at least consistent :P